src/manifolds/Elliptope.jl

```diff
@@ -0,0 +1,203 @@
+@doc raw"""
+    Elliptope{N,K} <: AbstractEmbeddedManifold{ℝ,DefaultIsometricEmbeddingType}
+
+The Elliptope manifold, also known as the set of correlation matricesmanifold of symmetric positive definite matrices, i.e.
+
+````math
+\begin{aligned}
+\mathcal E(n,k) =
+\bigl\{p ∈ ℝ^{n × n}\ \big|\ &a^\mathrm{T}pa \geq 0 \text{ for all } a ∈ ℝ^{n},\\
+&p_{ii} = 1 \text{ for all } i=1,\ldots,n,\\
+&\text{and } p = qq^{\mathrm{T}} \text{ for } q \in  ℝ^{n × k} \text{ with } \operatorname{rank}(p) = \operatorname{rank}(q) = k
+\bigr\}.
+\end{aligned}
+````
+
+And this manifold is working solely on the matrices $q$. Note that this $q$ is not unique,
+indeed for any orthogonal matrix $A$ we have $(qA)(qA)^{\mathrm{T}} = qq^{\mathrm{T}} = p$,
+so the manifold implemented here is the quotient manifold. The unit diagonal translates to
+unit norm columns of $q$.
+
+The tangent space at $p$, $T_p\mathcal E(n,k)$ also represented matrices $Y\in ℝ^{n × k}$ reads as
```

Comment:
```suggestion
The tangent space at $p$, denoted $T_p\mathcal E(n,k)$, is also represented by matrices $Y\in ℝ^{n × k}$:
```
If you want you can write it differently but the original sentence doesn't look right.